tests/evals/docker-compose.yml

```
@@ -0,0 +1,28 @@
+    command: gemini --yolo
+    volumes:
+      - ~/.gemini:/home/node/.gemini
+      - .:/workspace/mcp-server-evals
+    working_dir: /workspace/mcp-server-evals
+    stdin_open: true
+    tty: true
+    depends_on:
+      mcp-server:
+        condition: service_healthy
+    environment:
+      - MCP_SERVER_URL=http://mcp-server:8080 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)